pkg/servicescm/servicescm.go

```diff
@@ -0,0 +1,21 @@
+package servicescm
+
+import (
+	"fmt"
+
+	"github.com/openshift/windows-machine-config-operator/version"
+)
+
+// ServicesConfigMapName is the name of the ConfigMap detailing service configuration for a specific WMCO version
+var ServicesConfigMapName string
```

Comment:
This can just be `Name`, given the use sites will have this as `servicescm.Name`